mods/cnc/maps/nod07a/nod07a.lua

```diff
@@ -0,0 +1,267 @@
+			end
+		end)
+	end)
+end
+
+SetReinforcementsWaypoints = function(actor, waypoints)
+	if not actor.IsDead then
+		Utils.Do(waypoints, function(waypoint)
+			actor.Move(waypoint.Location)
+		end)
+		IdleHunt(actor)
+	end
+end
+
+StartWaves = function(Waves)
+	SendWaves(1, Waves)
+end
+
+Trigger.OnEnteredFootprint(AbandonedBaseTrigger, function(a, id)
+	if not abandonedBaseTrigger and a.Owner == player then
+		abandonedBaseTrigger = true
+
+		FlareCamera = Actor.Create("camera", true, { Owner = player, Location = waypoint25.Location })
+		Flare = Actor.Create("flare", true, { Owner = player, Location = waypoint25.Location })
+
+		Utils.Do(NodBase, function(actor)
+			if not actor.IsDead then
+				actor.Owner = player
+			end
+		end)
+
+		player.MarkCompletedObjective(NodObjective1)
+
+		Trigger.AfterDelay(DateTime.Seconds(3), function()
+			Media.PlaySpeechNotification(player, "NewOptions")
+		end)
+	end
+end)
+
+Trigger.OnEnteredFootprint(ReinforcementsTrigger, function(a, id)
+	if not reinforcementsTrigger and a.Owner == player and a.Type ~= 'harv' then
+		reinforcementsTrigger = true
+
+		Trigger.AfterDelay(DateTime.Seconds(5), function()
+			Media.PlaySpeechNotification(player, "Reinforce")
+			Reinforcements.ReinforceWithTransport(player, 'tran.in', ReinforcementsRockets, { ReinforcementsHelicopterSpawn.Location, waypoint24.Location }, { ReinforcementsHelicopterSpawn.Location }, nil, nil)
+		end)
+
+		StartWaves(IntroAttackWaves)
+
+		Trigger.AfterDelay(AirstrikeDelay, function() SendGDIAirstrike(GDIHQ, AirstrikeDelay) end)
+
+		Trigger.AfterDelay(DateTime.Minutes(2), function() ProduceInfantry(GDIPyle) end)
+
+		Trigger.AfterDelay(DateTime.Minutes(3), function() ProduceVehicle(GDIWeap) end)
+
+		Trigger.AfterDelay(DateTime.Minutes(3), function()StartWaves(AutoAttackWaves) end)
+
+		Trigger.AfterDelay(DateTime.Minutes(2), function()
+			Flare.Destroy()
+			FlareCamera.Kill()
+		end)
+	end
+end)
+
+WorldLoaded = function()
+	player = Player.GetPlayer("Nod")
+	enemy = Player.GetPlayer("GDI")
+	Camera.Position = waypoint26.CenterPosition
+
+	InsertNodUnits()
+	StartAI(GDICYard)
+
+	Trigger.OnObjectiveAdded(player, function(p, id)
+		Media.DisplayMessage(p.GetObjectiveDescription(id), "New " .. string.lower(p.GetObjectiveType(id)) .. " objective")
+	end)
+
+	Trigger.OnObjectiveCompleted(player, function(p, id)
+		Media.DisplayMessage(p.GetObjectiveDescription(id), "Objective completed")
+	end)
+
+	Trigger.OnObjectiveFailed(player, function(p, id)
+		Media.DisplayMessage(p.GetObjectiveDescription(id), "Objective failed")
+	end)
+
+	Trigger.OnPlayerWon(player, function()
+		Media.PlaySpeechNotification(player, "Win")
+	end)
+
+	Trigger.OnPlayerLost(player, function()
+		Media.PlaySpeechNotification(player, "Lose")
+	end)
+
+	NodObjective1 = player.AddPrimaryObjective("Find the Nod base.")
+	NodObjective2 = player.AddPrimaryObjective("Eliminate all GDI forces in the area.")
+	NodObjective3 = player.AddSecondaryObjective("Build 3 SAMs.")
```

Comment:
Maybe add:

> ... to fend of the GDI bombers.

as there is otherwise no indication in-game about them and their status.